packages/documentation/src/stories/getting-started/styles-primeng/primeng-theme-import.sample.scss

```diff
@@ -0,0 +1,7 @@
+@use "@swisspost/design-system-styles-primeng/primeng-theme";
+@use "primeng/resources/primeng.css";
+
+@layer design-system-styles, primeng;
+
+// refer to the @swisspost/design-system-styles installation guidelines to know which bundle-name and extension to choose for your project
+@import '@swisspost/design-system-styles/<bundle-name>.<extension>' layer(design-system-styles);
```

Comment:
Are we sure with this? Putting all the Design System Styles in a layer in an existing project could have unexpected consequences in random places where project styles suddenly are able to override Design System Styles (as they should in the future, but that's not the case right now).

The instruction also uses `@import` vs the Styles instructions use `@use`.